### PR TITLE
SW-5417: Remove makeStyles everywhere in Terraware web (#6)

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR removes uses of the deprecated `makeStyles` API.

NOTE: Will break changes for this ticket out into multiple PRs, with this being the **sixth** (and FINAL PR 🤞). Sorry this one is a bit larger than the others 😅

### CODE REVIEW

File: src/components/Map/MapRenderUtils.tsx [hunk 1]
```diff
@@ -1,88 +1,9 @@
...
-    '& .mapboxgl-popup-close-button': {
-      display: 'none',
-    },
-  },
-});
-
-/**
- * Species / plants renderer
- */
-export function useSpeciesPlantsRenderer(subzonesWithPlants: any): MapPopupRenderer {
```
Comment (constanzauanini) [inline src/components/Map/MapRenderUtils.tsx:50]: Is this not used?

**Review by constanzauanini — APPROVED** [review]
This was a big one! Lgtm!

File: src/components/Map/MapRenderUtils.tsx (hunk 1 above)
Comment (joshbuchea) [reply src/components/Map/MapRenderUtils.tsx:50]: Surprisingly no, it was only referenced in one other file where it was just being imported and re-exported

**Comment by joshbuchea** [comment]
Looks like there's a conflict now, will fix

**Pull request merged**